tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Jasmine/JasmineTest.html

```diff
@@ -0,0 +1,90 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title>Jasmine</title>
+    <link rel="shortcut icon" type="image/png" href="lib/jasmine-2.4.1/jasmine_favicon.png" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"></script>
+    <script src="http://matterwebapp.azurewebsites.net/lib/jasmine-2.4.1/jasmine.js"></script>
+    <link href="http://matterwebapp.azurewebsites.net/lib/jasmine-2.4.1/jasmine.css" rel="stylesheet" />
```

Comment:
remove absolute path https://matterwebapp.azurewebsites.net from src. The site may be deployed any and location and should developers should able to run from there local machine.